extensions/exposition/documentation/query.md

```diff
@@ -224,3 +224,46 @@ A list of Entity properties to be included in the Observation result.
 ```yaml
 projection: [id, title, timestamp]
 ```
+
+## Optimistic concurrency control
+
+If an operation returns an object with `_version` property,
+then its value is passed as the value of
+the [`etag` header](https://datatracker.ietf.org/doc/html/rfc7232#section-2.3) in the response
+(and removed from the object).
+
+Client can use the `if-match` request header to perform an operation only if the corresponding
+object has not been modified since the last retrieval.
```

Comment:
Any reason why we don't want to support `If-None-Match` also? Particularly `*` support there for idempotency

Reply:
Can you please provide a little bit more explanation and an example.

Reply:
Example with `*` from MDN: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-None-Match#sect1

Reply:
It's not like I don't understand the `*` syntax, I don't understand the functional meaning of `if-none-match` for managing entity's state.

Here is an example for `if-match`:

Moderator opens user profile:

```
GET /users/1/

---

200 OK
etag: "7"

name: "John"
```

Moderator marks the profile as `verified` ensuring that it hasn't changed since been loaded:

```http
PATCH /users/1
if-match: "7"

verified: true
```

What it the corresponding example for `if-none-match`?

Reply:
I don'y have a real immediate use case for that and theoretical cases may not make the same sense. This is mostly for ability doing `PUT` for a new resource without having to check previously if they already exists or sort of `one-by-one` processing. It's not a blocker anyway, so let's merge the implemented functionality.